packages/python/accern_xyme/accern_xyme.py

```diff
@@ -1583,7 +1650,8 @@ def render(value: str) -> Optional[str]:
 
                     import subprocess
                     cmd = ["echo", graph_str]
-                    p1 = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+                    p1 = subprocess.Popen(cmd, stdout=subprocess.PIPE) \
```

Comment:
actually with would be the correct way here -- is that a blocker? then add a FIXME